examples/demo-rollup/src/common.rs

```diff
@@ -1,124 +0,0 @@
-#[cfg(feature = "experimental")]
-use std::str::FromStr;
-
-use anyhow::Context as _;
-use demo_stf::runtime::Runtime;
-#[cfg(feature = "experimental")]
-use secp256k1::SecretKey;
-#[cfg(feature = "experimental")]
-use sov_cli::wallet_state::PrivateKeyAndAddress;
-use sov_db::ledger_db::LedgerDB;
-#[cfg(feature = "experimental")]
-use sov_ethereum::experimental::EthRpcConfig;
-#[cfg(feature = "experimental")]
-use sov_ethereum::GasPriceOracleConfig;
-use sov_modules_api::default_context::DefaultContext;
-#[cfg(feature = "experimental")]
-use sov_modules_api::default_signature::private_key::DefaultPrivateKey;
-use sov_modules_api::Spec;
-use sov_modules_stf_template::{SequencerOutcome, TxEffect};
-use sov_rollup_interface::da::DaSpec;
-use sov_rollup_interface::services::batch_builder::BatchBuilder;
-use sov_rollup_interface::services::da::DaService;
-use sov_sequencer::batch_builder::FiFoStrictBatchBuilder;
-use sov_sequencer::get_sequencer_rpc;
-use sov_state::ProverStorage;
-
-#[cfg(feature = "experimental")]
-const TX_SIGNER_PRIV_KEY_PATH: &str = "../test-data/keys/tx_signer_private_key.json";
-
-pub(crate) fn create_rpc_methods<Da: DaService + Clone>(
```

Comment:
The contents of this function are mostly duplicated in `mock_rollup.rs` and `celestia_rollup.rs` within their own `create_rpc_methods` implementations, which are identical. Should we maybe keep this around?

Reply:
The common code was moved to `sov_modules_rollup_template::register_rpc`.

Reply:
The common code was moved to `sov_modules_rollup_template::register_rpc`.

Reply:
👍